guides/shipments/shipping.md

```diff
@@ -1,129 +1,194 @@
 
-This guide explains how Solidus represents shipping options and how it calculates expected costs, and shows how you can configure the system with your own shipping methods. After reading it you should know:
+If your store has complex shipping needs, you may find one of Solidus's existing
+shipping extensions, like [`solidus_active_shipping`][solidus-active-shipping]
+or [`solidus_shipstation`][solidus-shipstation], useful. Check out [the list of
+supported Solidus Extensions](https://extensions.solidus.io).
 
-* how shipments and shipping are implemented in Solidus
-* how to specify your shipping structure
-* how split shipments work
-* how to configure products for special shipping treatment
-* how to capture shipping instructions
+This article provides a summary of shipping concepts. If you are interested in
+reading about example Solidus shipment setups see
+[Shipment setup examples](shipment-setup-examples.html.markdown).
 
-Solidus uses a very flexible and effective system to calculate shipping, accommodating the full range of shipment pricing: from simple flat rate to complex product-type- and weight-dependent calculations.
+<!-- TODO:
+  Add section that summarizes what Spree::Objects are created related to
+  shipments and explains what their function is in the larger checkout process.
+-->
 
-The Shipment model is used to track how items are delivered to the buyer.
+## Shipment attributes
 
-Shipments have the following attributes:
+The `Spree::Shipment` model tracks how items should be delivered to the
+customer.
 
-* `number`: The unique identifier for this shipment. It begins with the letter H and ends in an 11-digit number. This number is shown to the users, and can be used to find the order by calling `Spree::Shipment.find_by(number: number)`.
-* `tracking`: The identifier given for the shipping provider (i.e. FedEx, UPS, etc).
-* `shipped_at`: The time when the shipment was shipped.
-* `state`: The current state of the shipment.
-* `stock_location_id`: The ID of the Stock Location where the items for this shipment will be sourced from.
-* Other attributes likely of interest to developers:
-  * `adjustment_total`
-  * `additional_tax_total`
-  * `promo_total`
-  * `included_tax_total`
-  * `cost`
-  * `order_id`
+### Basic attributes
 
-**Needed:** shipment process flow diagram
+- `number`: The unique identifier for this shipment. It begins with the letter
+  `H` and ends in an 11-digit number. This number is visible to customers, and
+  it can be used to find the order (by calling `Spree::Shipment.find_by(number:
+  H12345678910)`).
+- `tracking`: The identifier given for the shipping provider (such as FedEx or
+  UPS).
+- `shipped_at`: The time when the shipment is shipped.
+- `state`: The current state of the shipment.
+- `stock_location_id`: The ID of the stock location where the items for this
+  shipment are sourced from.
 
-An explanation of the different shipment states:
+### Additional attributes
```

Comment:
I don't think this distinction makes sense. `cost` and `order_id` are just as important as `stock_location_id`.